code/game/objects/machinery/mortar.dm

```diff
@@ -373,6 +396,26 @@
 		return
 	return ..()
 
+// Artillery cameras. Together with the artillery impact hud tablet, shows a live feed of imapcts.
+
+/obj/machinery/camera/artillery
+	name = "artillery camera"
+	network = list("terragovartillery")
+	icon_state = "nothing" //you shouldn't be able to see this!
+	internal_light = FALSE
+	c_tag = "impact camera"
+	resistance_flags = RESIST_ALL
+	var/obj/machinery/deployable/mortar/artillery = null
+
+/obj/machinery/camera/artillery/update_icon()
+	icon_state = "nothing"
+
+/obj/machinery/camera/artillery/Destroy()
+	if(artillery)
+		artillery.impact_cam = null
+	return ..()
```

Comment:
i dont get why this subtype exists

Reply:
-Regular cameras are visible, we want this one to be invisible
-Regular cameras are not invincible. If cameras aren't invincible, they get destroyed by the mortar explosion which ends the feed too early
-If the camera is deleted we need to clean the var in the mortar (I think)
-A unique c_tag makes this UI easier to distinghish from other camera consoles (but could be set on mortar init())
-Internal_light needs to be set to false, otherwise AI can magically light the camera up
-Network needs to be set to terragovartillery so we have a unique camera net for this (because we don't want colony camera consoles, etc. being able to tap into this)

if there's a cleaner way of handling it, that'd be a better idea, though!

Reply:
artillery var is completely unused
update_icon should not be overriden